src/components/ApiRomainLevel2.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Css/ThemeApi.css';
+import { tokenApi } from '../.vscode/constants';
+
+function Level2() {
+  const [imageList, setImageList] = React.useState([]);
+  React.useEffect(() => {
+    fetch(`https://api.unsplash.com/search/photos/?client_id=${tokenApi}&query=Roman%20Empire&orientation=portrait&per_page=4`)
+      .then((response) => response.json())
+      .then((data) => {
+        setImageList([...data.results, ...data.results]);
+      });
+  }, []);
+  const shuffledImageList = [...imageList];
+  shuffledImageList.sort(() => Math.random() - 0.5);
+
+  return (
+    <>
+      {
+         shuffledImageList.map((image) => (
+           <img className="card" src={image.urls.thumb} alt="" />
+         ))
+     }
+    </>
+  );
+}
+
+export default Level2;
```

Comment:
it's the same component as Level1 with only one variation: the amount of images taken from the API. use a variable (through props) and make a unique component Level